src/reference/v8.json

```diff
@@ -97,6 +97,32 @@
         }
       }
     },
+    "state": {
+      "type": "state",
+      "default": {},
+      "doc": "The initial state of the style. This object where keys are property names and value a [`schema`](https://maplibre.org/maplibre-style-spec/schema) definition. State property value can be retrieved with the [`global-state`](https://maplibre.org/maplibre-style-spec/expressions/#global-state) expression.\n\n State is required when `global-state` is used in the style",
```

Comment:
I'm not sure this is well explained. is "schema definition" a new type? should it be in the types.md file instead of its own file?
Also this is not written well I believe: "...and value a schema definition."